tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/DeleteRecursive.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.tools.contentgenerator.cli;
+
+import static org.projectnessie.model.Content.Type.NAMESPACE;
+
+import com.google.common.base.Preconditions;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Stream;
+import org.projectnessie.client.api.CommitMultipleOperationsBuilder;
+import org.projectnessie.client.api.NessieApiV2;
+import org.projectnessie.error.NessieConflictException;
+import org.projectnessie.error.NessieNotFoundException;
+import org.projectnessie.model.Branch;
+import org.projectnessie.model.ContentKey;
+import org.projectnessie.model.EntriesResponse.Entry;
+import org.projectnessie.model.Operation.Delete;
+import org.projectnessie.model.Reference;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** Deletes a namespace recursively. */
+@Command(
+    name = "delete-recursive",
+    mixinStandardHelpOptions = true,
+    description = "Delete selected namespace and its nested objects (recursively)")
```

Comment:
```suggestion
    description = "Delete selected namespace and its child namespaces (recursively)")
```

Reply:
Hmmm, "nested objects" also includes tables :thinking:

Reply:
Nevermind, I thought that tables would never be deleted by this tool, which is why I thought that "nested objects" was rather deceiving.